python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -271,70 +191,30 @@ def _func_build_rf(
             verbose=verbose,
             rows_sample=rows_sample,
             max_leaves=max_leaves,
-            n_streams=n_streams,
             quantile_per_tree=quantile_per_tree,
-            seed=seed
-        )
+            dtype=dtype)
 
     @staticmethod
-    def _fit(model, input_data, convert_dtype):
-        X = concatenate([item[0] for item in input_data])
-        y = concatenate([item[1] for item in input_data])
-        return model.fit(X, y, convert_dtype)
+    def _func_build_rf(
```

Comment:
Is there a naming convention that decides whether to start with _func? I think you're using it for static functions, but I don't think it's universally on all statics... maybe just `_construct_rf`?